README.rst

```diff
@@ -73,7 +73,7 @@ Use a vector store to store embedded data and perform vector search.
 
 .. code-block:: python
 
-    from langchain_google_sapnner import SpannerVectorstore
+    from langchain_google_spanner import SpannerVectorstore
```

Comment:
Please make sure you keep your PRs scoped to 1 change at a time.

Reply:
Alright, reverted.